src/CSI.IBTA.Shared/Settings/ImagesSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace CSI.IBTA.Shared.Settings
+{
+    public static class ImagesSettings
+    {
+        public static string[] GetAllowedFormats()
```

Comment:
Suggestion is to extract file formats to the settings file, so the formats are configurable. You may do it in a separate PR,  this one could be approved when you done with other notes.